eva/udfs/abstract_udfs.py

```diff
@@ -67,3 +67,42 @@ def classify(self, frames: np.ndarray) -> pd.DataFrame:
 
     def __call__(self, *args, **kwargs):
         return self.classify(*args, **kwargs)
+
+
+class AbstractFilterUDF(metaclass=ABCMeta):
+    """
+    Abstract class for UDFs. All the UDFs which apply filters
```

Comment:
Can we not call these filters? Someone might think of them as filtering data

Reply:
What should I call them? Effects?